### ISSUE TITLE
TODOs in docstrings should be preserved

### ISSUE THREAD

**Issue by keith** [nicklockwood/SwiftFormat#959]
With this code:

```swift
    /// blah
    /// blah
    /// TODO: re-write as a flow: PPG-5401
    ///
    /// blah
```

SwiftFormat replaces the `TODO` line to only have 2 slashes

**Comment by nicklockwood** [comment]
@keith that's deliberate. Xcode doesn't correctly parse TODO: comments with three slashes

**Comment by keith** [comment]
Interesting, it is a weird trade-off since it being reformatted like that breaks the docstring

**Comment by nicklockwood** [comment]
Yes, maybe it should have a heuristic to avoid applying the change in the middle of a docstring block like that.

**Comment by nicklockwood** [comment]
@keith fixed in 0.48.6
